cypress/integration/login.js

```diff
@@ -20,5 +21,6 @@ describe('Login functionality', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User')
         cy.get('.main-nav').find('li').should('have.length', 1)
+        cy.get('[class^="calendar__day calendar--today calendar--selected"]').contains(today.getDate())
```

Comment:
calendar--selected - this shouldn't be required as you can select a different slot and the selector should still work for our test :)

Reply:
If the requirement is to verify that the current date is marked and selected then calendar--selected is required otherwise you are right.